client/my-sites/email/email-management/email-home.tsx

```diff
@@ -105,7 +105,7 @@ const EmailHome = ( props: EmailManagementHomeProps ) => {
 		return canCurrentUser( state, selectedSite.ID, 'manage_options' );
 	} );
 	const hasSitesLoaded = useSelector( hasLoadedSites );
-	const isAllDomainManagementContext = context === 'domains';
+	const isAllDomainManagementContext = context === 'domains' || context === 'hosting-overview';
```

Comment:
Hey 👋 This part is already handled under https://github.com/Automattic/wp-calypso/pull/98152, as that's already approved, should we leave it to be handled there?

Reply:
Hey @Imran92 ,

This part is the same, but one is missing from your PR #98152.

https://github.com/Automattic/wp-calypso/pull/98227/files#diff-c09ac2a6897a0f768fcc09a50158e5c57926fdf1a0cffd531403c2755d72f823R104

So we need it as well in order to show it properly for the `hosting-overview` context.